### PR TITLE
Fix decimal parsing using CultureInfo.InvariantCulture, fixes #68

### LINKED ISSUE #68: Failing tests related to decimal parsing & CurrentCulture
Having CultureInfo.CurrentCulture: "{es-ES}"

- Parlot.Tests.FluentTests.SwitchShouldReturnCommonType fails with:
    * `Assert.Equal() Failure Expected: 123.456 Actual: 123,456`

- Parlot.Tests.Calc.InterpreterTests.TestDecimalNumber fails with:
    * `Assert.Equal() Failure Expected: 123 Actual: 1230`
    * `Assert.Equal() Failure Expected: 123,1 Actual: 1231`
    * `Assert.Equal() Failure Expected: 123,456789 Actual: 123456789`

- Parlot.Tests.Calc.ParserTests.TestDecimalNumber fails with:
    * `Assert.Equal() Failure Expected: 123 Actual: 1230`
    * `Assert.Equal() Failure Expected: 123,1 Actual: 1231`
    * `Assert.Equal() Failure Expected: 123,456789 Actual: 123456789`

### PR DESCRIPTION
**Pull request by aviciano**
This PR fixes #68.

### CODE REVIEW

**Comment by sebastienros** [comment]
Can't find the button to trigger the build :/

**Review by lahma — APPROVED** [review]
Testing approve

**Comment by lahma** [comment]
GitHub has issues, https://www.githubstatus.com/

File: test/Parlot.Tests/Calc/Interpreter.cs [hunk 1]
```diff
@@ -1,5 +1,7 @@
 namespace Parlot.Tests.Calc
 {
+    using System.Globalization;
```
Comment (hishamco) [inline test/Parlot.Tests/Calc/Interpreter.cs:3]: Move the using above the namespace

File: test/Parlot.Tests/Calc/Parser.cs [hunk 2]
```diff
@@ -1,5 +1,7 @@
 namespace Parlot.Tests.Calc
 {
+    using System.Globalization;
```
Comment (hishamco) [inline test/Parlot.Tests/Calc/Parser.cs:3]: Move the using above the namespace

**Review by hishamco — CHANGES_REQUESTED** [review]
LGTM except a tiny comment

**Comment by hishamco** [comment]
@aviciano can you close and re-open the PR to trigger the build

**Comment by hishamco** [comment]
Seems everything is fine, I think it's ready to merge

**Pull request merged**